app.py

```diff
@@ -33,13 +34,14 @@ def post_validation(self, content):
 
 
 class RestaurantEndpoint(MethodView):
-    
     def __init__(self, restaurants):
         self.restaurants = restaurants
 
+    @swag_from("yaml\\restaurants.yml")
     def get(self):
         return jsonify(self.restaurants), 200
 
+    @swag_from("yaml\\restaurants.yml")
     def post(self):
         content = request.json
         validation.post_validation(content)
```

Comment:
what I observed:
ID should not be sent in POST, it is the application responsibility to set ID

Reply:
What ID?

Reply:
`POST` accepts a restaurant item to save, item has field `id`. but it is service's responsibility to assign the `id`, because `id` is system-level field

Reply:
fixed